### ISSUE TITLE
What the lib "pointops" for

### ISSUE THREAD

**Issue by RyanG41** [Pointcept/SegmentAnything3D#32]
Hi,
I'm kind of a rookie here. Could someone please tell me something about the lib "pointops"? I searched it on google but got insufficient results. I guess it is some tool for point operations and saw "pointops_cuda" when searching, but I wonder where is its source?
Thanks a lot.

**Comment by yhyang-myron** [comment]
Sorry for not understanding you well. There are some pointclouds operations implemented on cuda in the pointops. What do you mean **source**?

**Comment by RyanG41** [comment]
Sorry for not express the question well. What I mean is, is the pointclouds operations implemented by 'pointops' defined by individuals (i.e. author yourself), or is is some kind of library that was predefined such that we only need to call them? If it is a library stuff, where is it originated?

By the way, I follow the guidelines and run the code, but I dont know why the output pointcloud by SAM3D is somehow lack of groups(or labels? colors?). The final visualization of scene0000_00 is shown below. The running feedback of the terminal is normal(showing 0.jpg to 5560.jpg and merging pointcloud by half the number until it goes to 1). Do you happen to know where could be wrong?
<img width="1280" alt="result" src="https://github.com/Pointcept/SegmentAnything3D/assets/134327716/e3f10da7-6764-4eef-a4de-0e172989df48">
And the visualization code I use is:
`from util import *
import torch

# Define the file paths
pcd_filepath = 'F:/sam3d/processed_pcd/train/scene0000_00.pth'
pcd_seg_filepath = 'F:/sam3d/results/pcd/scene0000_00.pth'

# Load point cloud data
pcd_data = torch.load(pcd_filepath)
seg_data = torch.load(pcd_seg_filepath)

save_path = 'F:/sam3d/results/pcd/scene0000_00.ply'
visualize_partition(pcd_data['coord'], dict(group=seg_data)['group'], save_path)`
